rpm/gmp-droid.spec

```diff
@@ -24,9 +28,14 @@ Gecko Media Plugin for droidmedia codec support in Gecko based browsers
 %install
 %meson_install
 
+# create config oneshot
+mkdir -p $RPM_BUILD_ROOT%{_oneshotdir}
```

Comment:
`install -D` already creates all the necessary directories in the path.

Reply:
I had to add the full filepath for that to work, but it works. Thanks.